Gemfile

```diff
@@ -26,7 +26,7 @@ group :test do
     gem "aruba", "~> 0.14"
     gem "capybara"
     gem "nokogiri", RUBY_VERSION < "2.1" ? "~> 1.6.0" : ">= 1.7"
-    gem "cucumber"
+    gem "cucumber", "> 2"
```

Comment:
I guess you meant to say `< 3` ?

Reply:
👍 ... let me update and bring in the exclude pattern so we know we have successful tests before merging... I originally had ```>~2``` and made a change to get travis to run again.

Reply:
already included it in the rubocop PR